grakn-test/src/test/java/ai/grakn/test/migration/json/JsonMigratorMainTest.java

```diff
@@ -65,12 +65,12 @@ public void setup() {
 
     @Test
     public void jsonMigratorMainTest(){
-        runAndAssertDataCorrect("-input", dataFile, "-template", templateFile, "-keyspace", graph.getKeyspace());
+        runAndAssertDataCorrect("-u", engine.uri(), "-input", dataFile, "-template", templateFile, "-keyspace", graph.getKeyspace());
     }
 
     @Test
     public void jsonMainDistributedLoaderTest(){
-        runAndAssertDataCorrect("-input", dataFile, "-template", templateFile, "-keyspace", graph.getKeyspace(), "-uri", "localhost:4567");
```

Comment:
This test has changed to the extent that it doesn't really test anything. Any idea on how to resolve this?